### DISCUSSION TITLE
Status of MODEL in v5.0 release?

### DISCUSSION THREAD

**Discussion by GregIthaca** [ge-high-assurance/RACK#314] [Q&A]
I see that v5.0 dropped a couple of hours ago.  I've pulled the docker image and the git repo, and from what I can tell, it looks like Issue #299 is not yet integrated.  I was hoping someone could let me know whether we (as TA1s) can assume the present v5.0 is stable from now until the expected software delivery, or if I should be looking for a new v5.1 or v5-dev that contains the proposed MODEL changes?

Thank you,
Greg Nelson (GrammaTech, Inc.)

**Comment by kityansiu** [comment]
MODEL did not make it into v5.0, unfortunately. The review from the Data Model Decision Team (aka, Jedi Council) was not completed before our v5.0 ontology freeze date. We do have some items planned for v5.1, which includes the MODEL ontology extension and ingestion of add'l Boeing platform data. 

@davearcher & @glguy, please feel free to comment.

**Reply by GregIthaca** [reply]
I see that this branch was now merged into master.  Should I be tying myself to the tagged v5.0 for the upcoming release instead of to master, or does this mean a v5.1 is about to drop?

**Reply by kityansiu** [reply]
I think the best thing for you to do is use v5.0 for the upcoming assessment. Our target time for v5.1 is end of March.
